scripts/run_examples_dev

```diff
@@ -7,37 +7,56 @@ source "$SCRIPTS_DIR/common"
 # Build Oak server.
 "$SCRIPTS_DIR/build_server_dev" "$@"
 
-# Run Oak server.
-./bazel-clang-bin/oak/server/dev/oak &
-SERVER_PID=$!
 # TODO: Ensure that background processes are killed with something like `trap cleanup_fn EXIT`.
 
 # Run Rust-based Oak examples against a common Oak server instance.
-# Note that the chat example cannot be run because the client is interactive.
-# TODO(#462): Re-enable chat example, even if it may be just loading the server-side code and not
-# interact with it.
-readonly RUST_EXAMPLES="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/rust$' | cut -d'/' -f2 | uniq | grep -v chat)"
+readonly RUST_EXAMPLES="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/rust$' | cut -d'/' -f2 | uniq)"
+
```

Comment:
Is this script mostly a copy of `run_examples`, but with a different runner? If that's literally the only difference, I guess they should be refactored to share most of the logic. Perhaps create a separate issue if it's not practical to do in this one?

Reply:
Created an issue: https://github.com/project-oak/oak/issues/595